servicetest/filesystem/test_filesystem.py

```diff
@@ -251,15 +251,94 @@ def test_tmpfs_writebuffer_size_enabled(self):
             os.remove(absoluteTestFile)
 
     @pytest.mark.xfail("platform.release() <= \"3.18.0\"")
-    def test_tmpfs_writebuffer_size_disabled(self):
+    def test_tmpfs_writebuffer_size_enabled_10mb(self):
         """ The SoftwareContainer will create a tmpfs in the location of the
             containers temporary directory location if the write buffer is
             enabled. Test that the tmpfs mounted in the containers temporary
             directory structure works as expected.
 
-            Note: The filesystem is hard coded to 100MB at the time this test
-            was written, but will be configurable via the DATA field in a
-            short while
+            This test will create a 10 MB tmpfs which will be tested that it
+            behaves as expected.
```

Comment:
Files larger than 10MB can not be written and files smaller than 9MB can be written :). I'll add it.